core/src/main/java/lucee/commons/lang/ParserString.java

```diff
@@ -700,6 +711,42 @@ public boolean removeSpace() {
 		}
 		return (start < pos);
 	}
+	/**
+	 * Strip out all sql comments
+	 * 
+	 * @return SQL text with comments stripped out
+	 */
+	public String stripSqlComments(String sql) {
+		char c;
+		int sqlLen = sql.length();
+		StringBuilder sb = new StringBuilder();
+
+		for (int i = 0; i < sqlLen; i++) {
+			c = sql.charAt(i);
+			if ( i < (sqlLen - 1)) {
+				// handle multi line comment
+				if (c == '/' && sql.charAt(i + 1) == '*') {
+					int end = sql.indexOf("*/", i + 2);
```

Comment:
yoz don't handle at all if the multi line or single line comment is within a literal string like
`select '/* multi */  // -- single' as comments`